### PR TITLE
[ZH] Translation of 03-3

### PR DESCRIPTION
**Pull request by cyjj**

### CODE REVIEW

File: docs/zh/week03/03-3.md [hunk 1]
```diff
@@ -4,5 +4,200 @@ lang-ref: ch.03-3
 title: Properties of natural signals
 authors: Ashwin Bhola, Nyutian Long, Linfeng Zhang, and Poornima Haridas
 date: 15 February 2020
+translator: Matt Tso
```
Comment (Atcold) [inline docs/zh/week03/03-3.md]: translation-date: 20 Mar 2020

File: docs/zh/week03/03-3.md [hunk 2]
```diff
@@ -4,5 +4,201 @@ lang-ref: ch.03-3
 title: Properties of natural signals
```
Comment (Atcold) [inline docs/zh/week03/03-3.md]: The title is not translated!
Send a PR with this last bit, plz.
Just the title translation, not everything again.

Comment (cyjj) [reply docs/zh/week03/03-3.md]: sry for forgetting the title. What's the meaning of not everything again?
and whether translator name attached? I found someone attached someone not. a little bit confused.

Comment (Atcold) [reply docs/zh/week03/03-3.md:4]: Yes, add translator, translation-date, and title in Chinese.
Not everything, meaning you can send a PR with just a few lines edit. Do not submit the whole translation again. Just can also just click on "Edit" directly on GitHub, and change things there.

**Pull request merged**